decisions.md

```diff
@@ -2,23 +2,43 @@
 layout: default
 ---
 
-## The Good Docs Project Decisions
+# Decisions
 
-### Criteria for Decision Making
+This page describes the decision making processes used within The Good Docs Project.
+
+**Version:** 2.0
+
+**Last updated:** August 2020
+
+## Voting
 
 Decisions are based on voting using the following:
 
-    +1 : For
-    +0: Mildly for, but mostly indifferent
-    0: Indifferent
-    -0: Mildly against, but mostly indifferent
-    -1 : Strongly against. This is expected to be accompanied by a reason.
+*   +1 : For
+*   +0: Mildly for, but mostly indifferent
+*   0: Indifferent
+*   -0: Mildly against, but mostly indifferent
+*   -1 : Strongly against. This is expected to be accompanied by a reason.
```

Comment:
I feel like this is an odd scale to use, because +/-0 is relatively obtuse. Perhaps +2/+1/0/-1/-2 is more intuitive?

Reply:
I agree with @Loquacity on this point. Otherwise this scale is fine.

Reply:
I agree with @Loquacity on this point too.

Reply:
The scale as proposed is standard within the open source community (and makes it very easy to tally up the results) so I'd prefer it stays as it is.

Reply:
If the original scale is standard in open source then I suppose we ought to go with that. I withdraw my objection.

Reply:
I have never seen this scale used before, honestly. How do you tally up results effectively if you use 0 to mean three different things?

Reply:
This scale was inherited from the Open Source Geospatial Foundation, by all the projects and committees that I'm aware of. I'm pretty sure it was in turn inherited from Apache communities. I'm in favor of following community conventions where possible. If we can find a widely used alternative best practice, I'm happy to follow.

Reply:
I've raise a bug to track this, and suggest it be addressed in a future release. I don't see it as a show stopper for getting these processes out.
#46